site/viamguides/src/securibot/securibot.md

```diff
@@ -0,0 +1,318 @@
+1. **Save** your updates, and confirm `YES` to apply OS customization settings. Confirm `YES` to erase data on your storage device. You may also be prompted by your operating system to enter an administrator password. After granting permissions to the Imager, it will begin writing and then verifying the Linux installation to your storage device.
+1. Remove the storage device from your computer when the installation is complete.
+
+### Connect with SSH
+
+1. Place the storage device into your Raspberry Pi and boot the Pi by plugging it in to an outlet. A red LED will turn on to indicate that the Pi is connected to power.
+   > aside negative
+   > Make sure you are using a 5V 3A USB-C power supply to power your Raspberry Pi 4. Using a power supply with inadequate amperage can lead to instability, throttling, or unexpected behavior. Additionally, while USB boot is enabled by default on newer Raspberry Pi 4 models, older versions may require a firmware update to enable it. Refer to the [Raspberry Pi documentation](https://www.raspberrypi.com/documentation/computers/raspberry-pi.html) for detailed setup instructions and compatibility.
+1. Once the Pi is started, connect to it with SSH. From a command line terminal window, enter the following command. The text in <> should be replaced (including the < and > symbols themselves) with the user and hostname you configured when you set up your Pi.
+   ```bash
+   ssh <USERNAME>@<HOSTNAME>.local
+   ```
+1. If you are prompted “Are you sure you want to continue connecting?”, type “yes” and hit enter. Then, enter the password for your username. You should be greeted by a login message and a command prompt.
+   ![raspberry pi SSH login](assets/sshLogin.webp)
+1. Update your Raspberry Pi to ensure all the latest packages are installed
+   ```bash
+   sudo apt update
+   sudo apt upgrade
+   ```
+
+<!-- ------------------------ -->
+
+## Set up the hardware
+
+Duration: 5
+
+The Viam office currently uses a fob system to unlock a security door. By adding computer vision and facial recognition, we can verify that specific individuals are authorized to enter and automatically unlock the door. In this setup, a webcam is mounted above the door to detect faces. When an authorized face is recognized, a servo waves a heated pad in front of a PIR motion sensor (originally intended for people leaving the office) to trigger the door to unlock.
+
+Your door setup may differ, so this section may need to be adjusted based on your specific configuration.
+
+### Add your servo
+
+1. **Review the wiring diagram**: The servo can be controlled via a GPIO pin on the Raspberry Pi. Refer to the following wiring diagram to see how to connect the Raspberry Pi to the servo, using the breadboard and also resistors to control the flow of electricity. The breadboard simplifies the wiring process by providing a secure platform to connect components without soldering.
```

Comment:
What breadboard is this referring to? I don't see one in the wiring diagram or any of the pictures. Same with the reference to resistors.